lib/mastodon/domains_cli.rb

```diff
@@ -26,6 +27,12 @@ def self.exit_on_failure?
       When the --limited-federation-mode option is given, instead of purging accounts
       from a single domain, all accounts from domains that have not been explicitly allowed
       are removed from the database.
+
+      When the --by-uri option is given, DOMAIN is used to match the domain part of actor
+      URIs rather than the domain part of the webfinger handle. For instance, an account
+      that has the handle `foo@bar.com` but whose profile is at the URL
+      `https://mastodon-bar.com/users/foo`, would be purged by either
```

Comment:
Do you mean . rather than - here? mastodon-bar.com should probably not be matched by any action on bar.com

Reply:
Never mind, this is a tricky case but it makes sense now.